projects/1.16/assets/tips/tips/lang/zh_cn.json

```
@@ -1 +1,72 @@
-{}
\ No newline at end of file
+{
+  "tips.title.tip": "提示",
+  "tips.tip.no_tips": "未加载提示，请检查配置。",
+  "tips.tip.undead_damage": "像骷髅这样的亡灵生物会被治疗药水伤害，会被伤害药水治疗。",
+  "tips.tip.poison_immune": "僵尸、骷髅和蜘蛛免疫剧毒药水。",
+  "tips.tip.witch_conversion": "被闪电击中的村民将变成女巫。",
+  "tips.tip.four_eggs": "抛出鸡蛋时，有极小概率生成4只小鸡。",
+  "tips.tip.nether_lava": "熔岩在下界的流动速度会更快。",
+  "tips.tip.east": "太阳、月亮和星星都向东方移动。",
+  "tips.tip.swamp_slime": "史莱姆会在满月时的沼泽中生成。",
+  "tips.tip.deflect_ghast_fireballs": "你能够通过攻击恶魂发出的火焰弹来改变它的运动方向。",
```

```suggestion
  "tips.tip.deflect_ghast_fireballs": "攻击恶魂火球可以改变它的运动方向。",
```
[恶魂/火球 - Minecraft Wiki，最详细的我的世界百科](https://minecraft.fandom.com/zh/wiki/%E6%81%B6%E9%AD%82?so=search#.E7.81.AB.E7.90.83)